### ISSUE TITLE
make clean does not clean artifacts generated by make rpm

### ISSUE THREAD

**Issue by laggarcia** [kimchi-project/ginger#11]
After running make rpm, if you run make clean to clean all the build artifacts and get a clean source tree again, you'll still have files left from the make rpm process.

Steps to reproduce:

1) make rpm
2) make clean
3) git status
On branch master
Your branch is up-to-date with 'origin/master'.
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   po/de_DE.po
	modified:   po/en_US.po
	modified:   po/es_ES.po
	modified:   po/fr_FR.po
	modified:   po/ginger.pot
	modified:   po/it_IT.po
	modified:   po/ja_JP.po
	modified:   po/ko_KR.po
	modified:   po/pt_BR.po
	modified:   po/ru_RU.po
	modified:   po/zh_CN.po
	modified:   po/zh_TW.po

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	build-aux/compile
	po/de_DE.gmo
	po/en_US.gmo
	po/es_ES.gmo
	po/fr_FR.gmo
	po/it_IT.gmo
	po/ja_JP.gmo
	po/ko_KR.gmo
	po/pt_BR.gmo
	po/ru_RU.gmo
	po/stamp-po
	po/zh_CN.gmo
	po/zh_TW.gmo

no changes added to commit (use "git add" and/or "git commit -a")
[laggarcia@lagarcia ginger]$

**Comment by danielhb** [comment]
I've tried to reproduce the behavior with a fresh ginger source tree and this is what I saw:

$ git status
On branch master
Your branch is up-to-date with 'origin/master'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   po/en_US.po
	modified:   po/ginger.pot
	modified:   po/pt_BR.po
	modified:   po/zh_CN.po

no changes added to commit (use "git add" and/or "git commit -a")


These changes are related to outdated .po files that I'll commit upstream. However I don't have the same amount of languages that you have. Also I haven't seen the .gmo and build-aux/compile in the 'untracked' session. 

Care to test again with a fresh ginger tree?
